lms/envs/common.py

```diff
@@ -2312,6 +2312,9 @@ def _make_locale_paths(settings):
     'poll',
 
     'xblock_jupyter_graded',
+
+    # Navoica API
+    'navoica_api',
```

Comment:
Jeśli dodajesz navoica_api, a nie znajduje się ono jeszcze w requirementsach to należy dodać biblioteki które muszą być przeinstalowane podczas update.
w miejscach requirements/edx/base.txt i requirements/edx/development.txt
-e git+https://github.com/OPI-PIB/navoica-api.git#egg=navoica-api
(porządek alfabetyczny)